Sources/SwiftInspectorVisitors/PropertyVisitor.swift

```diff
@@ -168,12 +168,30 @@ public struct PropertyInfo: Codable, Hashable, CustomDebugStringConvertible {
     }
   }
 
+  public enum Paradigm {
```

Comment:
Dan—Francisco and I spoke further and aligned on this approach rather than including the nodes themselves in `PropertyInfo`. This approach allows `PropertyInfo` to remain a `Codable` struct. Moreover, we are able to capture some of the code information into this enum.

Reply:
oooh I like this!